joomla/joomla-cms
# [6.2] Feature/collapseable tables

Pull Request for Issue #38335 .

### Summary of Changes

Added function for collapsing child item for:
- Menu List View
- Categories List View (used in Category, Newsfeed, Banners)
- Tags List View

### Testing Instructions

1. Go to one of the above listed views and create multiple Items with child items in different Levels.
2. Set the listlimmit to All
3. After apply the patch
4. Click to the arrow Buttons in the Table row. The Child items should disapaer and the arraw should trun from down to right
5. Click again to the arrow button, The Child Items should again apear and the arrow turns from right to dowm

### Actual result BEFORE applying this Pull Request

No function, because new feature

### Expected result AFTER applying this Pull Request

New feature should availible. How it works is described in the Testing Instruction

### Link to documentations
Please select:
